src/editor/Collaboration.js

```diff
@@ -12,14 +12,23 @@ define(function (require, exports, module) {
             // immediately ask for camera access
             autoRequestMedia: false
         });
+        //To be moved to the bramble API.
+        var hash = window.location.hash.replace(/^#/, "");
+        var m = /&?collaboration=([^&]*)/.exec(hash);
```

Comment:
this regex looks wrong, `window.location.hash` returns a fragment id not the querystring. Your regex seems to be dealing with querystrings. If it is meant to be on the querystring, do something like:
```js
var query = (new URL(window.location.href)).searchParams;
this.room = query.collaboration || <random string>;
```